src.ts/utils/format.ts

```diff
@@ -0,0 +1,37 @@
+const BN = require ('bn.js');
+
+// Exponent for calculating how many units of account balance are in one near.
+const exp = 24;
+// actual number of units of account balance in one near.
+const unitsInOneNear = new BN('10', 10).pow(new BN(exp, 10));
+
+/**
+ * Convert account balance to near.
+ * @param balance
+ */
+export function convertAccountBalanceAmountToNear(balance: string): string {
+    const amtBN = new BN(balance, 10);
+    if (amtBN.lte(unitsInOneNear)) {
+        return `0.${balance.padStart(exp, '0')}`;
+    }
+    return `${amtBN.div(unitsInOneNear).toString(10, 0)}.${amtBN.mod(unitsInOneNear).toString(10, 0)}`;
+}
+
+/**
+ *
+ * @param amt
+ */
+export function nearAmountToAccountBalanceAmount(amt) {
```

Comment:
`convertAccountBalanceAmountToNear` and `nearAmountToAccountBalanceAmount` look confusing to me. Why one of them is `convert` and another isn't? Feels less confusing if we stick to `format` for one direction and `parse` for another. E.g. `parseBalance` and `formatBalance`.

Reply:
Balance might not be the best option (e.g. for amount to transfer from one acct to another). Maybe parseNearAmount and formatNearAmount?